### PR TITLE
extract geth command, remove childchain only code paths

### PR DESCRIPTION
**Pull request by InoMurko**
Cleanup of Watcher code by removing some of  the childchain only code and tests + simplifying PG DB configuration.

### CODE REVIEW

**Comment by InoMurko** [comment]
@pnowosie I would kindly ask you to look at the tests I deleted/modified. I think they're childchain specific. If they're not and I deleted too much, let me know.

File: config/releases.exs [hunk 1]
```diff
@@ -5,7 +5,21 @@ import Config
 #
 # See https://hexdocs.pm/mix/1.9.0/Mix.Tasks.Release.html#module-runtime-configuration
 
+mandatory = fn
+  env, exception, true, true ->
```
Comment (pnowosie) [inline config/releases.exs:9]:
```suggestion
  env_var_name, error_msg, true, true ->
```
Also please add a comment what these parameters are.

**Review by pnowosie — APPROVED** [review]
:+1:

File: docker-compose.yml [hunk 2]
```diff
@@ -21,34 +21,9 @@ services:
...
-            --keystore=./data/geth/keystore/ \
-            --password /tmp/geth-blank-password \
-            --unlock "0,1" \
-            --rpc --rpcapi personal,web3,eth,net --rpcaddr 0.0.0.0 --rpcvhosts=* --rpcport=8545 \
-            --ws --wsaddr 0.0.0.0 --wsorigins='*' \
-            --mine \
-            --allow-insecure-unlock
+    entrypoint: /bin/sh -c ". data/command"
+    environment:
+      RPC_PORT: 8545
```
Comment (ayrat555) [inline docker-compose.yml:26]: Does it need to be configurable? 8545 is the default value. And I can find places where another port is used

**Review by ayrat555 — APPROVED** [review]

File: docker-compose.yml (hunk 2 above)
Comment (InoMurko) [reply docker-compose.yml:26]: not really, I took it from childchain and ... I prefer consistency!

**Pull request merged**